comb/compileCommands/lib.nix

```diff
@@ -0,0 +1,22 @@
+{
+  inputs,
+  cell
+}: {
+  compileCommands = pkg:
+    pkg.overrideAttrs (super: {
+      installPhase = "cp compile_commands.json $out";
+      cmakeFlags = super.cmakeFlags ++ ["-DCMAKE_EXPORT_COMPILE_COMMANDS=1"];
```

Comment:
I wonder how CMAKE_EXPORT_COMPILE_COMMANDS works in combination with cc-wrapper-hook.

I assume worst case scenario is duplicated commands in compile_commands.json but I hope it never runs cc-wrapper-hook.